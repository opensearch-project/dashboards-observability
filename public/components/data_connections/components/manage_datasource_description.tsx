/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */

import { EuiSpacer, EuiText, EuiTitle, EuiHorizontalRule } from '@elastic/eui';
import _ from 'lodash';
import React from 'react';

export function DataConnectionsDescription() {
  return (
    <div>
      <EuiTitle size="s">
        <h2>Manage existing data connections</h2>
      </EuiTitle>

      <EuiSpacer size="s" />
      <EuiText size="s" color="subdued">
        Manage already created data source connections.
      </EuiText>
      <EuiHorizontalRule size="full" />
    </div>
  );
}
