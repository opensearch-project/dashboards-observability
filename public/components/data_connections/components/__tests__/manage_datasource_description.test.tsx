/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */

import { configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { waitFor } from '@testing-library/react';
import React from 'react';
import { DataConnectionsDescription } from '../manage_datasource_description';

describe('Manage Datasource Description test', () => {
  configure({ adapter: new Adapter() });

  it('Renders manage datasource description', async () => {
    const wrapper = mount(<DataConnectionsDescription />);

    await waitFor(() => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
