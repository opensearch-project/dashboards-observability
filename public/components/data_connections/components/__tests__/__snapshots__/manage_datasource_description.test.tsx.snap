// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Manage Datasource Description test Renders manage datasource description 1`] = `
<DataConnectionsDescription>
  <div>
    <EuiTitle
      size="s"
    >
      <h2
        className="euiTitle euiTitle--small"
      >
        Manage existing data connections
      </h2>
    </EuiTitle>
    <EuiSpacer
      size="s"
    >
      <div
        className="euiSpacer euiSpacer--s"
      />
    </EuiSpacer>
    <EuiText
      color="subdued"
      size="s"
    >
      <div
        className="euiText euiText--small"
      >
        <EuiTextColor
          color="subdued"
          component="div"
        >
          <div
            className="euiTextColor euiTextColor--subdued"
          >
            Manage already created data source connections.
          </div>
        </EuiTextColor>
      </div>
    </EuiText>
    <EuiHorizontalRule
      size="full"
    >
      <hr
        className="euiHorizontalRule euiHorizontalRule--full euiHorizontalRule--marginLarge"
      />
    </EuiHorizontalRule>
  </div>
</DataConnectionsDescription>
`;
